e2e/integration/spec/authoring-api/hooks.spec.js

```diff
@@ -0,0 +1,78 @@
+const client = require('../../utils/client');
+
+const keyPath = 'path/to/key';
```

Comment:
I think it's a bit problematic to have state between tests. (as we might run them in parallel in future)

Reply:
@Yshayy I agree, I initially spent several hours setting up something that updates the hooks file in the git repo and creates a unique state for each test.
It worked as far as updating the actual git repo goes, but it seems the local git repo on the authoring service didn't pick up the updates and so the authoring api still had an outdated version of the git repo.

Reply:
I didn't want to go with something too messy like manually modifying it there as well, so if you have ideas on a decent way to do it I'm open to suggestions

Reply:
I will make the changes we discussed